cmake/MiNiFiOptions.cmake

```diff
@@ -82,6 +80,8 @@ list(APPEND STRICT_GSL_CHECKS_Values AUDIT ON DEBUG_ONLY OFF)
 set_property(CACHE STRICT_GSL_CHECKS PROPERTY STRINGS ${STRICT_GSL_CHECKS_Values})
 
 if (WIN32)
+    add_minifi_option(INSTALLER_MERGE_MODULES "Creates installer with merge modules" OFF)
+    add_minifi_option(INSTALLER_WITH_VC_REDISTRIBUTABLES "Creates installer with Visual C++ redistributables included" OFF)
```

Comment:
Could we add the MINIFI_ prefix to these new options?
(In my opinion we should add that to every option thats ours so it doesnt conflict with thirdparty options and it makes it easier to filter out thirdparty related options (e.g. python bootstrap).)

So adding it to new flags should be good way to start this transition.

Reply:
Updated in b2da5ed65397525433093fa654edfe0d7836ca13